web/src/app/shared/components/private-public-project/private-public-project.component.ts

```diff
@@ -40,7 +40,8 @@ export class PrivatePublicProjectComponent implements OnInit, OnDestroy {
     if (this.router.url === '/') {
       this.projectSubscription = this.projectService
         .findPublicProjects()
-        .subscribe((projects: IProject[]) => this.projects = projects.map((project: IProject) => new ProjectModel(project)));
+        .subscribe((projects: IProject[]) => this.projects = projects.map((project: IProject) => new ProjectModel(project))
+          .filter((project: ProjectModel) => project.repositories.length > 1));
```

Comment:
`> 1`, that will be `2` or more? We need it to be `1` or more, so `> 0` would be better?